cucumber/policy/features/support/world.rb

```diff
@@ -73,7 +73,9 @@ def admin_api_key
   end
 
   def admin_password
-    ENV['CONJUR_AUTHN_PASSWORD'] || 'admin'
+    ENV['CONJUR_AUTHN_API_KEY'] || begin
+      raise StandardError.new('Environment variable `CONJUR_AUTHN_API_KEY` must be set.')
+    end
```

Comment:
Maybe you took my comment too literally. :)

I was thinking something more along the lines of

`raise 'Provide CONJUR_AUTHN_API_KEY' unless ENV['CONJUR_AUTHN_API_KEY'].present?`

Either way is fine, though.

Reply:
I was originally thinking what you were thinking, then I started googling.... You know where that ends 😀